dev-scripts/test-cdc-lifecycle-e2e.py

```
@@ -0,0 +1,718 @@
+            head_status == get_status == 200 and len(body) > 0,
+            f"{label} NAR HEAD/GET agree (both 200, bytes present)",
+        )
+
+    # C. fsck repair-not-delete: fsck --repair must not destroy a NAR that a
+    #    narinfo still references. Reclaiming genuinely orphaned nar_file rows
+    #    (e.g. the transient xz/none duplicates the CDC on/off toggling leaves
+    #    behind) is correct fsck behavior, so a row-count DECREASE is expected
+    #    and not a failure. The real invariant is that every still-referenced
+    #    NAR keeps serving after the repair.
+    before = total_nar_count(db)
+    rc, out = fsck(db, storage, repair=True)
```

Ruff (0.15.15)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)